packages/protocol/src/vaults/borrowing/BorrowingVault.sol

```diff
@@ -24,15 +24,19 @@ import {
 } from "openzeppelin-contracts/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import {IVault} from "../../interfaces/IVault.sol";
 import {ILendingProvider} from "../../interfaces/ILendingProvider.sol";
+import {IHarvestManager, Strategy} from "../../interfaces/IHarvestManager.sol";
+import {IHarvestable} from "../../interfaces/IHarvestable.sol";
 import {IFujiOracle} from "../../interfaces/IFujiOracle.sol";
 import {SafeERC20} from "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
 import {Math} from "openzeppelin-contracts/contracts/utils/math/Math.sol";
 import {BaseVault} from "../../abstracts/BaseVault.sol";
 import {VaultPermissions} from "../VaultPermissions.sol";
+import {ISwapper} from "../../interfaces/ISwapper.sol";
 
 contract BorrowingVault is BaseVault {
   using Math for uint256;
   using SafeERC20 for IERC20Metadata;
+  using SafeERC20 for IERC20;
```

Comment:
Since IERC20Metadata is IERC20, just feel free to use IERC20Metadata. Same applies to upgradeable borrowing vault.